man/veritytab.xml

```diff
@@ -60,13 +60,61 @@ This is based on crypttab(5).
 
     <variablelist class='fstab-options'>
 
+      <varlistentry>
+        <term><option>no-superblock</option></term>
+
+        <listitem><para>Use dm-verity without permanent on-disk superblock.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>format=<replaceable>NUMBER</replaceable></option></term>
+
+        <listitem><para>Specifies the hash version type. Format type 0 is original Chrome OS version. Format type 1 is
+        current version.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>data-block-size=<replaceable>BYTES</replaceable></option></term>
+
+        <listitem><para>Used block size for the data device. (Note kernel supports only page-size as maximum
+        here.)</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>hash-block-size=<replaceable>BYTES</replaceable></option></term>
+
+        <listitem><para>Used block size for the hash device. (Note kernel supports only page-size as maximum
+        here.)</para></listitem>
```

Comment:
so what are possible values here? 512 and 4K effectively as I undertand?

Reply:
Looks to be multiple of 512.